ContextFactory/ConfiguredContextFactory.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace JMS\SerializerBundle\ContextFactory;
+
+use JMS\Serializer\Context;
+use JMS\Serializer\ContextFactory\DeserializationContextFactoryInterface;
+use JMS\Serializer\ContextFactory\SerializationContextFactoryInterface;
+use JMS\Serializer\DeserializationContext;
+use JMS\Serializer\SerializationContext;
+
+/**
+ * Class ConfiguredContextFactory
+ */
+class ConfiguredContextFactory implements SerializationContextFactoryInterface, DeserializationContextFactoryInterface
+{
+    /**
+     * Context config
+     *
+     * @var array
+     */
+    private $config;
+
+    /**
+     * ConfiguredContextFactory constructor.
+     *
+     * @param array $config Context configuration
+     */
+    public function __construct(array $config)
```

Comment:
never liked "magic" arrays of configurations... to buggy and you always have to read the implementation to know what happens. 

why not put into this class properties as `groups`, `version`, `serialize_nulls`, whit getters and setters, and configure them via dependency injection component?